cms/jinja2/templates/components/census/census-information.html

```diff
@@ -1,4 +1,4 @@
 <div class="ons-u-mb-xl census-2021">
-    {% include "templates/components/census/census-logo.html" %}
+    <img src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg" alt="Census 2021 logo" width="171" class="ons-u-mb-s">
```

Comment:
Removed the include here as we need a different class in this case.

Reply:
question: could we pass the class to the include?

Reply:
I tried that, but nunjucks doesn't have a `with` option for includes. I think we'd have to make it a macro, which feels like overkill.

Reply:
```jinja2
{% with census_logo_classes="ons-u-mb-s" %}
    {% include "templates/components/census/census-logo.html" %}
{% endwith %}
```

should do it.

Reply:
ooh hello.